msvc/scripts/csproj.tmpl

```diff
@@ -9,6 +9,9 @@
     <ProjectGuid>@PROJECTGUID@</ProjectGuid>
     <OutputType>@OUTPUTTYPE@</OutputType>
     <NoWarn>@DISABLEDWARNINGS@</NoWarn>
+    <HostPlatform Condition=" '$(HostPlatform)' == '' and '$(OS)' == 'Windows_NT'">win32</HostPlatform>

```

Comment:
This is to autodetect the current platform where msbuild is running on and use the corresponding configuration, basically like we do it in configure.ac. If you want to cross-compile you can override this by passing `/p:HostPlatform=darwin` to msbuild.